lib/pgBackRest/Common/Log.pm

```diff
@@ -763,4 +763,54 @@ sub testCheck
 
 push @EXPORT, qw(testCheck);
 
+####################################################################################################################################
+# logLevel - get the current log levels
+####################################################################################################################################
+sub logLevel
+{
+    return ($strLogLevelFile, $strLogLevelConsole, $strLogLevelStdErr, $bLogTimestamp);
+}
+
+push @EXPORT, qw(logLevel);
+
+####################################################################################################################################
+# logFileCacheClear - Clear the log file cache for testing
+####################################################################################################################################
+sub logFileCacheClear
+{
+    undef($strLogFileCache);
+}
+
+push @EXPORT, qw(logFileCacheClear);
+
+####################################################################################################################################
+# logFileCacheTestSimilar - Test for a string somewhere in the log file
+####################################################################################################################################
+sub logFileCacheTestSimilar
+{
+    my $strTest = shift;
+
+    my $strTmpLogFileCache = $strLogFileCache;
+
+    # Strip all whitespace
+    $strTmpLogFileCache =~ s/\s+//g;
+    $strTest =~ s/\s+//g;
+
+    return (($strTmpLogFileCache =~ m/$strTest/) ? true : false);
+}
+
+push @EXPORT, qw(logFileCacheTestSimilar);
+
+####################################################################################################################################
+# logFileCacheTestExact - Test for an exact match of the log
+####################################################################################################################################
+sub logFileCacheTestExact
+{
+    my $strTest = shift;
+
+    return (($strLogFileCache eq $strTest) ? true : false);
+}
+
+push @EXPORT, qw(logFileCacheTestExact);
+
```

Comment:
Both of these functions should be moved the run test and in Log.pm there should just be a function called `logFileCacheClear` to return the log cache.  I'd like to keep test code out of production to the maximum extent possible.